lib/test/test_hamburger.rb

```diff
@@ -65,7 +65,6 @@ def test_level_teacher_en
     assert_equal visibility[:show_student_options],     Hamburger::HIDE_ALWAYS
     assert_equal visibility[:show_signed_out_options],  Hamburger::HIDE_ALWAYS
     assert_equal visibility[:show_pegasus_options],     Hamburger::SHOW_ALWAYS
-    assert_equal visibility[:show_help_options],        Hamburger::SHOW_ALWAYS
```

Comment:
can these instead of being removed, be moved to a new `lib/test/test_help_header.rb` test suite?

Reply:
These visibility ones are probably not necessary to be tests (the help options are always shown) but I've pulled out the tests deleted below into a test file (and moved some helper code around).